### ISSUE TITLE
SVG stroke-linejoin attribute - arcs and miter-clip values need separate data points

### ISSUE THREAD

**Issue by chrisdavidmills** [mdn/browser-compat-data#25336]
### What type of issue is this?

Missing compatibility data

### What information was incorrect, unhelpful, or incomplete?

In the https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linejoin page, the descriptions of the `arcs` and `miter-clip` values say that they were introduced in SVG2 and aren't widely supported yet". I'm about to merge a PR that corrects the typo on those notes and removes the link to the BCD table (it is less than useless when there is no BCD data specific to these values), but it struck me that I should file an issue so that maybe we can add this data at some point.

After the data is added to the BCD, we can probably eliminate those notes from the content page altogether.

### What did you expect to see?

Data for the `arcs` and `miter-clip` values.

### Did you test this? If so, how?

I looked at the content page.

**Comment by caugner** [comment]
For the **CSS** property [stroke-linejoin](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-linejoin#browser_compatibility), BCD currently has sub-features for the values `bevel`, `miter`, and `round`, while the CSS spec also includes `crop`, `arcs`, and `fallback`.

For the **SVG** attribute [stroke-linejoin](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/stroke-linejoin#browser_compatibility), BCD currently has no sub-features (like most other SVG attributes), while the SVG spec defines values `miter`, `miter-clip`, `round`, `bevel`, and `arcs`.

Looking at [the example on the MDN page](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/stroke-linejoin#example), it seems that the SVG attribute also supports only `miter`, `round`, and `bevel` in Chrome (136), Firefox (138) and Safari 18.4, while `miter-clip` and `arcs` fallback to (i.e. look the same as) `miter`.

Note that there are stale tracking bugs for `stroke-linejoin: arcs` (only [Chrome](https://issues.chromium.org/issues/40362409) and [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1239142)) , and none for `miter-clip`.

tl;dr There is no intent to implement for `stroke-linejoin: arcs/miter-clip`, which would be required for inclusion in BCD.

I'm closing this issue as won't fix, but I'm bringing up SVG attribute subfeatures in one of the next BCD meetings.

**Comment by caugner** [comment]
> I'm closing this issue as won't fix, but I'm bringing up SVG attribute subfeatures in one of the next BCD meetings.

To follow-up: We discussed this in yesterday's BCD meeting, and since SVG presentational attributes map to CSS properties, they support the same values as the equivalent CSS property, so we don't see compelling reasons to duplicate the data here.

In this case, the SVG spec mentions additional values `arcs`, and `miter-clip`, but these are not implemented by any browser, and are marked "at risk" since https://github.com/w3c/svgwg/issues/592.
